src/main/scala/org/phenoscape/kb/Main.scala

```diff
@@ -117,741 +117,743 @@ object Main extends HttpApp with App {
   def routes: Route = cors() {
     alwaysCache(memoryCache, cacheKeyer) {
       respondWithHeaders(RawHeader("Vary", "negotiate, Accept")) {
-        pathSingleSlash {
-          redirect(Uri("http://kb.phenoscape.org/apidocs/"), StatusCodes.SeeOther)
-        } ~ pathPrefix("kb") {
-          path("annotation_summary") {
-            complete {
-              KB.annotationSummary
-            }
-          } ~
-            path("annotation_report") {
+        rejectEmptyResponse {
```

Comment:
@Shalsh23 this line (120) is the only real change.

Reply:
Okay, thanks.